transforms/class.js

```diff
@@ -1193,20 +1204,14 @@ module.exports = (file, api, options) => {
           isInitialStateConvertible(childPath) &&
           canConvertToClass(childPath)
         ) {
-          result.didTransform = true;
+          didTransform = true;
           updateToClass(childPath);
         } else {
-          result.didFallback = true;
+          didFallback = true;
           fallbackToCreateClassModule(childPath);
         }
       });
-      return result;
-    };
-
-    const {
-      didTransform,
-      didFallback,
-    } = apply(ReactUtils.findAllReactCreateClassCalls(root));
```

Comment:
FWIW this was the only use of `findAllReactCreateClassCalls` util method in the repo